data/json/requirements/toolsets.json

```diff
@@ -1,5 +1,23 @@
 [
   {
+    "id": "cooking_standard",
+    "type": "requirement",
+    "//": "Cooking of standard meals",
+    "qualities": [ { "id": "COOK", "level": 2 } ],
+    "tools": [ [
+      [ "fire", -1 ],
+      [ "hotplate", 10 ],
+      [ "mess_kit", 10 ],
+      [ "mil_mess_kit", 10 ],
+      [ "oil_cooker", 10 ],
+      [ "survivor_mess_kit", 10 ],
+      [ "gasoline_cooker", 10 ],
+      [ "charcoal_cooker", 10 ],
+      [ "popcan_stove", 10 ],
+      [ "toolset", 10 ]
+    ] ]
+  },
```

Comment:
> Its  KEY_MINUS  to resume or repeat the last crafting task?

That key works perfectly fine when the crafting has been suspended, as it happens when you answer the "You feel tired, stop crafting?" query with yes. After a good night sleep, the character gets up, I press the "repeat crafting" key and the crafting *continues* - the message logs explicitly states "You resume your task."

For this to works the activity must be suspendable (`player_activity::is_suspendable`). It also works when you abort the crafting (press the pause key while crafting).